tools/runners/run-all.js

```diff
@@ -70,11 +70,18 @@ class Runner {
       proxyToHost: appHost,
       onFailure
     });
+    
+    buildmessage.capture(function () {
+      self.projectContext.resolveConstraints();
+    });
+
+    const packageMap = self.projectContext.packageMap;
+    const hasMongoDevServerPackage = packageMap && packageMap.getInfo('mongo-dev-server') != null;
```

Comment:
Would you mind switching to use `!==` just to make sure we're matching the rest of the source.